bindings/jupyroot/python/JupyROOT/helpers/utils.py

```diff
@@ -454,13 +475,20 @@ class NotebookDrawer(object):
 
     def __init__(self, theObject):
         self.drawableObject = theObject
-        self.isCanvas = self.drawableObject.ClassName() == "TCanvas"
+        self.isRCanvas = False
```

Comment:
Just to be more concise, I'd just do `self.isRCanvas = hasattr(self.drawableObject,"ResolveSharedPtrs")` here and leave the `self.isCanvas` assignment as it was.

Reply:
RCanvas does not have `ClassName()` method